templates/master/00-master/alibabacloud/files/opt-libexec-openshift-alibabacloud-routes-sh.yaml

```diff
@@ -0,0 +1,178 @@
+mode: 0755
+path: "/opt/libexec/openshift-alibabacloud-routes.sh"
+contents:
+  inline: |
+    #!/bin/bash
+
+    # Prevent hairpin traffic when the apiserver is up
+
+    # As per the Alibaba Cloud documentation (https://www.alibabacloud.com/help/doc-detail/55206.htm),
+    # if a backend is load-balanced to itself, then the traffic will be dropped.
```

Comment:
It would be interesting to add the AlibabaCloud's reference on the `kube-apiserver-watcher` [README](https://github.com/openshift/machine-config-operator/tree/master/cmd/apiserver-watcher#background)

Reply:
I have added a section for Alibaba Cloud to the README.